### ISSUE TITLE
Typo in Riverside Medical Clinic `iss` value

### ISSUE THREAD

**Issue by msinjin** [the-commons-project/vci-directory#741]
The vci-issuers.json file has a typo in the `iss` value for Riverside Medical Clinic.

The domain "sf1.rmcps.com" is not in the correct case based on observation of production samples.

The following entry:

```json
    {
      "iss": "https://sf1.rmcps.com/FHIRProxy/api/epic/2021/Security/Open/EcKeys/32001/SHC",
      "name": "Riverside Medical Clinic",
      "website": "https://www.riversidemedicalclinic.com/"
    },
```

should be changed to:

```json
   {
      "iss": "https://SF1.rmcps.com/FHIRProxy/api/epic/2021/Security/Open/EcKeys/32001/SHC",
      "name": "Riverside Medical Clinic",
      "website": "https://www.riversidemedicalclinic.com/"
    },
```

**Comment by isaacvetter** [comment]
Hey @msinjin ,

I think that domain names are case-insensitive. For example, see: https://datatracker.ietf.org/doc/html/rfc4343#section-2

It looks like the path of Riverside's iss url is the same between the directory and your observation of production. Is that the case?

Isaac

**Comment by msinjin** [comment]
Hi @isaacvetter

> I think that domain names are case-insensitive. For example, see: https://datatracker.ietf.org/doc/html/rfc4343#section-2

That's correct. However, the fhir spec is case-sensitive, so one can run into trouble when matching VCI iss values to those in the QR payloads. 

It isn't a big problem. For my purpose I added an extra step to casefold both iss values before comparison. Others doing similar work might not catch the difference, though, and fail silently. 

> It looks like the path of Riverside's iss url is the same between the directory and your observation of production. Is that the case?

That is correct, the iss url is a match (other than case) between your list and those I've seen in the wild. The same is true for the other two Issues I've posted reporting similar case mismatches. 

Note, of millions of processed SHC samples from production across all current issuers in your list, only the three I reported were not perfect, case-sensitive matches for the iss values in the QR payloads. Pretty good! 🙂

Best, 
Mark
